key-vault.tf

```diff
@@ -0,0 +1,14 @@
+module "vault" {
+  source                  = "git@github.com:hmcts/cnp-module-key-vault?ref=master"
+  name                    = "${var.product}-${var.env}"
+  product                 = "${var.product}"
+  env                     = "${var.env}"
+  tenant_id               = "${var.tenant_id}"
+  object_id               = "${var.jenkins_AAD_objectId}"
+  resource_group_name     = "${azurerm_resource_group.rg.name}"
+  product_group_object_id = "70de400b-4f47-4f25-a4f0-45e1ee4e4ae3"
```

Comment:
Just for my knowledge how is this generated? Is this same for all the repos related to shared infra across reform ?

Reply:
This is the object id for the AAD group containing developers who should have write access. There's some documentation about how to retrieve it here https://github.com/hmcts/cnp-module-key-vault#product_group_object_id

Reply:
In this instance its - `dcd_reform_dev_logs` as we want anyone to be able to add secrets to ease onboarding of services